services/director/tests/test_registry_cache_task.py

```diff
@@ -46,7 +47,7 @@ async def test_registry_caching_task(loop, client, push_services):
         number_of_computational_services=1, number_of_interactive_services=1
     )
     # the services shall be updated
-    await sleep(config.DIRECTOR_REGISTRY_CACHING_TTL)
+    await sleep(config.DIRECTOR_REGISTRY_CACHING_TTL * 1.1) # NOTE: this can take some time. Sleep increased by 10%.
```

Comment:
MINOR: why not simply increasing the variable itself? Is it used somewhere else?

Reply:
yes it is used by the director registry caching task